JM/CF/Scripts/1_Core/CommunityFramework/Logging/CF_Trace.c

```diff
@@ -27,11 +28,11 @@ class CF_Trace : Managed
 		}
 		
 		string instanceName;
-		g_Script.CallFunction(instance, "GetDebugNameNative", instanceName, null);
-		if (instanceName == string.Empty)
-		{
+		//g_Script.CallFunction(instance, "GetDebugNameNative", instanceName, null);
+		//if (instanceName == string.Empty)
+		//{
 			instanceName = "" + instance;
-		}
+		//}
```

Comment:
Why not just wrap this in a `if(g_Script)` and or `if(instance)` to avoid nullptr problems when called and either of them is not available yet? Now we lack the GetDebugNameNative information even if it is available.

Reply:
It's not crashing because g_Script is null. It's crashing due to some change behind the scenes in 1.16 for `GetDebugNameNative`